operator/src/main/java/org/bf2/operator/operands/KafkaCluster.java

```diff
@@ -242,6 +250,29 @@ public Kafka kafkaFrom(ManagedKafka managedKafka, Kafka current) {
         return kafka;
     }
 
+    @Override
+    public int getReplicas(ManagedKafka managedKafka) {
+        return getBrokerReplicas(managedKafka, cachedKafka(managedKafka));
+    }
+
+    public int getBrokerReplicas(ManagedKafka managedKafka, Kafka current) {
+        Integer maxPartitions = managedKafka.getSpec().getCapacity().getMaxPartitions();
+        int scalingAndReplicationFactor = 3;
+        int desiredReplicas = scalingAndReplicationFactor;
+        if (maxPartitions != null) {
+            double physicalCapacity = config.getKafka().getPartitionCapacity();
+            desiredReplicas = (int) Math.ceil(maxPartitions * scalingAndReplicationFactor / physicalCapacity);
+            // round to only even multiples
+            desiredReplicas = (int) (Math.ceil(desiredReplicas / (double)scalingAndReplicationFactor) * scalingAndReplicationFactor);
```

Comment:
This comment isn't matching up with my parsing of the calculation. For the current configuration (assuming `1000` max partitions from control plane), I think this updates `desiredReplicas` from 2 to 3. What am I missing?

Reply:
> I think this updates desiredReplicas from 2 to 3. What am I missing?

I don't know that you are missing anything.  It will round from 2 to 3 as we only want even multiples of the scalingAndReplicationFactor.  If we determine that 4 brokers are needed, we'll then round to 6 and so forth.

Reply:
Ah, I was missing that we want even multiples "of the scalingAndReplicationFactor" rather than simply "even".